src/main.rs

```diff
@@ -119,11 +119,19 @@ impl Command for Disp {
     }
 }
 
+#[derive(Clone, PartialEq, Eq, PartialOrd, Ord, ValueEnum)]
```

Comment:
Are all of these `derive` arguments necessary for `clap`?

Reply:
This was a new suggestion from rust-analyzer. I actually don't think the Ord derives probably do not make sense. Though some might consider certain types better than others.